### ISSUE TITLE
Add `MoViNet` model

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#2304]
**Short Description**

**MoViNets: Mobile Video Networks for Efficient Video Recognition**

Mobile Video Networks (MoViNets) are efficient video classification models **runnable on mobile devices**. MoViNets demonstrate state-of-the-art accuracy and efficiency on several large-scale video action recognition datasets.

On [Kinetics 600](https://deepmind.com/research/open-source/kinetics), MoViNet-A6 achieves 84.8% top-1 accuracy, outperforming recent Vision Transformer models like [ViViT](https://arxiv.org/abs/2103.15691) (83.0%) and [VATT](https://arxiv.org/abs/2104.11178) (83.6%) without any additional training data, while using 10x fewer FLOPs. And **streaming** MoViNet-A0 achieves 72% accuracy while using 3x fewer FLOPs than MobileNetV3-large (68%).

**Papers**

[MoViNets](https://arxiv.org/abs/2103.11511)

**Existing Implementations**

- Official-TensorFlow (Keras 2):  [tf-models/projects/movinet](https://github.com/tensorflow/models/tree/master/official/projects/movinet)
- Unofficial pytorch: https://github.com/Atze00/MoViNet-pytorch/tree/main

**Other Information**

The streaming version of this model makes it quite impression and it would be valuable addition.

**Comment by divyashreepathihalli** [comment]
Hi @innat that you for this suggestion. We will keep this open, but at this point this is of low priority for the team.

**Comment by innat** [comment]
@divyashreepathihalli Thanks for the confirmation. I pulled out the movinet from tf-model garden and maintaining to a dedicated repo (private for now). The codebase somewhat complex due to large number of configurations. I will keep update the codebase, so, please let me know when keras-cv is ready take it.

**Comment by divyashreepathihalli** [comment]
If you have code ready to go, which works well across all backends. Please feel free to open the PR. We will review it and add it.

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new **KerasHub** package, which supports image, text, and multi-modal models. Please read [the announcement](https://github.com/keras-team/keras-hub/issues/1831). `KerasHub` will support all the core functionality of `KerasCV`.

`KerasHub` can be installed with `!pip install -U keras-hub`. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

**Comment by innat** [comment]
@sachinprasadhs 
> If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

Firstly, identifying which feature is required or missing can be done effectively, either by practitioners or the Keras team. Tickets in `keras-cv` were created only when such features were **NOT** available or when a specific need was encountered. Commenting after a significant delay and requesting the recreation of the same issue is neither efficient nor appropriate. Kindly move this ticket (and any other similar tickets with the same context) from `keras-cv` to `keras-hub`.
